src/Symfony/Component/Mailer/Transport/Smtp/EsmtpTransport.php

```diff
@@ -71,6 +71,14 @@ public function __construct(string $host = 'localhost', int $port = 0, ?bool $tl
         $stream->setPort($port);
     }
 
+    public function forceIpv4(): void
```

Comment:
what about not adding this method and moving the code inline into EsmtpTransportFactory?